andela/ah-backend-odin
# #161340194 Reporting an article

#### What does this PR do?
Users should be able to report an article

#### Description of Task to be completed?
As an authenticated user I should be able to report articles that violate terms of the agreement or are plagiarised to the authors haven admin for removal and or any other actions as per the terms of engagement

#### How should this be manually tested?
After cloning the repo to your machine, cd into it and run this snippet in the terminal
`python manage.py runserver`

Open your Postman or Postico and feed it with the following URL after having logged in and been authenticated.
`http://127.0.0.1:8000/api/articles/<slug>/report/`

The BODY of your POST request should contain the following content
`{ "report":
                    { "reason": "the reason as to why the article is being reported"} 
                }`

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#161340194

#### Screenshots (if appropriate)
#### Questions:
